### PR TITLE
First text added to main intro page

### PR DESCRIPTION
**Pull request by corinnebosley**
@marqh @bjlittle @dkillick

This is a first attempt at adding some text to the introduction on the homepage.  Please let me know if you can see some areas of improvement or some better wording, or whatever.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -4,9 +4,28 @@
...
+There are three different ways which you can use Cube Browser:
+
+1. Explorers:
+
+If you have several cubes which you would like to plot and explore in different ways, have a look at our `Explorers<explorers/intro.html>`_, which offer you a ready-made cube-to-plot selection process.
+These are available to download or to run straight from your internet browser.
+
+2. Browsing cubes:
+
+You may want to use the Cube Browser API to write a short python code which will make your plots, instead of using the Explorer.
```
Comment (marqh) [inline doc/index.rst]: I might replace this sentence with:
`You may want to use the Cube Browser API to write a Jupyter notebook to make a simple or complicated interactive visualisation.`

File: doc/index.rst [hunk 2]
```diff
@@ -4,9 +4,28 @@
...
+This will give you more flexibility in the way you would like to plot your data and how you would like it laid out and customized.
+To do this, you can use the `guide<browsing_cubes/info.html>`_ and `reference<ref_docs.html>`_ pages.
+
+3. Writing your own cube browser:
+
+In the same way that we have constructed our Explorers, you can make use of Ipywidgets to string together your own cube plotting process.
+We have laid out a set of instructions to take you step-by-step through `writing a cube browser<write_your_own/intro.html>`_,
```
Comment (marqh) [inline doc/index.rst]: maybe:
`We have provided some developer documentation illustrating how to use IPywidgets to build a work flow`

**Comment by corinnebosley** [comment]
Yes.  They sound better.

File: doc/index.rst [hunk 3]
```diff
@@ -4,9 +4,28 @@
...
+so you can make a work flow as general or as specialized as you need.
+This could be useful if you have a specific combination of plots which you use frequently and repeatedly.
```
Comment (marqh) [inline doc/index.rst]: I would also reference the explorer source code at this point

File: doc/index.rst [hunk 4]
```diff
@@ -4,9 +4,28 @@
...
+You may want to use the Cube Browser API to write a Jupyter notebook to make a simple or complicated interactive visualisation.
+This will give you more flexibility in the way you would like to plot your data and how you would like it laid out and customized.
+To do this, you can use the `guide<browsing_cubes/info.html>`_ and `reference<ref_docs.html>`_ pages.
+
+3. Writing your own cube browser:
+
+In the same way that we have constructed our Explorers, you can make use of Ipywidgets to string together your own cube plotting process.
+We have provided some `developer documentation<write_your_own/intro.html>`_ illustrating how to use IPywidgets to build a work flow as general or as specialized as you need.
+This could be useful if you have a specific combination of plots which you use frequently and repeatedly.
+`Click here<../lib/cube_browser/explorer.py>`_ to see the source code for the Explorer we have provided.
```
Comment (corinnebosley) [inline doc/index.rst]: @marqh This doesn't look right to me.  Firstly, I am not convinced that that is the right way to make a link to the source code, secondly, the sentence doesn't sit right with me.

**Comment by corinnebosley** [comment]
@marqh 
I am now happy with this iteration of the main docs page.  Unless you have any more suggestions I am keen to get this merged and move forward.

**Comment by corinnebosley** [comment]
Thank you

**Pull request merged**